Dockerfile

```diff
@@ -41,4 +41,4 @@ VOLUME ["/data"]
 EXPOSE 5000
```

Comment:
Please copy `devdata/index-v2.tinysearch` to somewhere in the image (`/data`?) for the moment. In long term, we may revisit https://github.com/mwmbl/mwmbl/issues/27 for a better workflow.

Reply:
Adding `ADD devdata/index-v2.tinysearch /app/devdata/` seems fine.